fabulaws/library/wsgiautoscale/api.py

```diff
@@ -746,13 +746,22 @@ def bootstrap(purge=False):
 @roles('web', 'worker')
 def update_source(changeset=None):
     """Checkout the latest code from repo."""
-    if changeset is None:
-        changeset = env.branch
     require('environment', provided_by=env.environments)
     with cd(env.code_root):
         sudo('find . -name "*.pyc" -delete')
-        vcs('pull')
-        vcs('update', [changeset])
+        if env.vcs_cmd.endswith('git'):
```

Comment:
This hg v. git stuff is all over the 'vcs' command.  otherwise I'd have forgotten about it too.